code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -417,7 +417,7 @@
 		active_program = null
 	var/mob/user = usr
 	if(user && istype(user))
-		ui_interact(user) // Re-open the UI on this computer. It should show the main screen now.
+		UNLINT(INVOKE_ASYNC(src, /obj/item/modular_computer/ui_interact, user)) // Re-open the UI on this computer. It should show the main screen now.
```

Comment:
What couldn't you figure out? Should at least be commented.

Reply:
Like, I'm pretty sure you can just `/datum/proc/ui_interact` and you're fine.

Reply:
byond doesn't like .proc/ui_interact, and it doesn't like /obj/item/modular_computer/proc/ui_interact, but smdmm doesn't like a proc path that goes nowhere

Reply:
well uh